packages/web/src/services/env/env.dev.ts

```diff
@@ -79,8 +79,8 @@ export const env: Env = {
   USER_NODE: 'http://audius-protocol-creator-node-1',
   USE_HASH_ROUTING: false,
   WAUDIO_MINT_ADDRESS: '37RCjhgV1qGV2Q54EHFScdxZ22ydRMdKMtVgod47fDP3',
-  WEB3_NETWORK_ID: '1337',
-  WEB3_PROVIDER_URL: 'http://audius-protocol-poa-ganache-1',
```

Comment:
where were these used? maybe we can remove entirely now since we go through the relay on discovery?